lib/specs/v2.js

```diff
@@ -498,6 +498,96 @@ let rules = {
+        details: oneLineTrim`The <code>.kg-bookmark-content</code> CSS class is required otherwise the bookmark card will appear unstyled.
+        Find out more about required theme changes for the Koenig editor <a href="${docsBaseUrl}editor/#bookmark-card" target=_blank>here</a>.`,
+        regex: /\.kg-bookmark-content/g,
+        className: '.kg-bookmark-content',
+        css: true
+    },
+    'GS050-CSS-KGBMTI': {
+        level: 'error',
+        rule: 'The <code>.kg-bookmark-title</code> CSS class is required to appear styled in your theme',
+        details: oneLineTrim`The <code>.kg-bookmark-title</code> CSS class is required otherwise the bookmark card will appear unstyled.
+        Find out more about required theme changes for the Koenig editor <a href="${docsBaseUrl}editor/#bookmark-card" target=_blank>here</a>.`,
+        regex: /\.kg-bookmark-title/g,
+        className: '.kg-bookmark-title',
+        css: true
+    },
+    'GS050-CSS-KGBMDE': {
+        level: 'error',
+        rule: 'The <code>.kg-bookmark-description</code> CSS class is required to appear styled in your theme',
+        details: oneLineTrim`The <code>.kg-bookmark-description</code> CSS class is required otherwise the bookmark card will appear unstyled.
+        Find out more about required theme changes for the Koenig editor <a href="${docsBaseUrl}editor/#bookmark-card" target=_blank>here</a>.`,
+        regex: /\.kg-bookmark-description/g,
+        className: '.kg-bookmark-description',
+        css: true
+    },
+    'GS050-CSS-KGBMME': {
+        level: 'error',
+        rule: 'The <code>.kg-bookmark-metadata</code> CSS class is required to appear styled in your theme',
+        details: oneLineTrim`The <code>.kg-bookmark-metadata</code> CSS class is required otherwise the bookmark card will appear unstyled.
+        Find out more about required theme changes for the Koenig editor <a href="${docsBaseUrl}editor/#bookmark-card" target=_blank>here</a>.`,
+        regex: /\.kg-bookmark-metadata/g,
+        className: '.kg-bookmark-metadata',
+        css: true
+    },
+    'GS050-CSS-KGBMIC': {
+        level: 'error',
+        rule: 'The <code>.kg-bookmark-icon</code> CSS class is required to appear styled in your theme',
+        details: oneLineTrim`The <code>.kg-bookmark-icon</code> CSS class is required otherwise the bookmark card will appear unstyled.
+        Find out more about required theme changes for the Koenig editor <a href="${docsBaseUrl}editor/#bookmark-card" target=_blank>here</a>.`,
+        regex: /\.kg-bookmark-icon/g,
+        className: '.kg-bookmark-icon',
+        css: true
+    },
+    'GS050-CSS-KGBMAU': {
+        level: 'error',
+        rule: 'The <code>.kg-bookmark-author</code> CSS class is required to appear styled in your theme',
+        details: oneLineTrim`The <code>.kg-bookmark-author</code> CSS class is required otherwise the bookmark card will appear unstyled.
```

Comment:
```suggestion
        details: oneLineTrim`The <code>.kg-bookmark-author</code> CSS class is required otherwise the bookmark card author name will appear unstyled.
```